autoload/lsp/signature.vim

```diff
@@ -55,15 +74,22 @@ export def BufferInit(lspserver: dict<any>)
     return
   endif
 
+  var triggerChars = lspserver.caps.signatureHelpProvider.triggerCharacters
+
   # map characters that trigger signature help
-  for ch in lspserver.caps.signatureHelpProvider.triggerCharacters
+  for ch in triggerChars
     var mapChar = ch
     if ch =~ ' '
       mapChar = '<Space>'
```

Comment:
The conversion of the space character to `<Space>` is needed to map the space character below using `inoremap`.
This is not needed if `InsertCharPre` is used to check for the character.

Reply:
Removed the portion